### PR TITLE
fix(frontend): add responsiveness to "last version" btn in files and html text

### PR DESCRIPTION
**Pull request by MariePrandstetter**
#4573 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

File: frontend_app_html-document/src/container/HtmlDocument.jsx [hunk 1]
```diff
@@ -909,12 +909,12 @@ export class HtmlDocument extends React.Component {
...
-                  className='wsContentGeneric__option__menu__lastversion html-document__lastversionbtn btn highlightBtn'
+                  className='wsContentGeneric__option__menu__lastversion html-document__lastversionbtn btn'
                   onClick={this.handleClickLastVersion}
                   style={{ backgroundColor: state.config.hexcolor, color: '#fdfdfd' }}
                 >
                   <i className='fas fa-history' />
-                  {props.t('Last version')}
+                  <span>{props.t('Last version')}</span>
                 </button>
               )}
```
Comment (raphj) [inline frontend_app_html-document/src/container/HtmlDocument.jsx:919]: We may want to open a technical dept issue  to remove the duplication in apps (duplicated here: https://github.com/tracim/tracim/pull/4670/files#diff-391722c17977af4e654df7a5c6f6b36798e660c0f821ab2910cbc7f8b70c3240R1141)

File: frontend_app_html-document/src/css/index.styl [hunk 2]
```diff
@@ -83,6 +83,7 @@
         &__content
           color darkGrey
       &__version
+      // MB - 21/05/2021 - This bit of code doesn't seem to be used anywhere
```
Comment (raphj) [inline frontend_app_html-document/src/css/index.styl:86]:
```suggestion
      // MB - 2021-05-21 - FIXME (#xxx)
      // This bit of code doesn't seem to be used anywhere
```

Can you open a technical dept issue for this and replace xxx by the id of this issue?

**Review by raphj — APPROVED** [review]

File: frontend_app_html-document/src/container/HtmlDocument.jsx (hunk 1 above)
Comment (MariePrandstetter) [reply frontend_app_html-document/src/container/HtmlDocument.jsx:919]: I agree, I think this specific repetition only happens twice though.

**Comment by gbonaspetti** [comment]
There are three things that I see during testing that I don't think are connected to this development, so I leave it to you to choose between making a new ticket for that or fixing it in this PR:
- The button icon is bigger than the other buttons icons
- The button does not have a cursor pointer
- The button does not have a tooltip

Note: if you decide to fix, I suggest to use IconButton component.

**Review by raphj — APPROVED** [review]

**Comment by charline-rageade** [comment]
Tested on branch

**Pull request merged**